c10/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+project(c10 LANGUAGES CXX)
+
+set(LIB_SOURCES
+        dummy.cpp
+)
+
+set(TEST_SOURCES
+        dummy_test.cpp
+)
+
+if("${CMAKE_CXX_COMPILER_ID}" MATCHES "MSVC")
+  # TODO Also add some warning options that MSVC can understand
+  set(WARNING_OPTIONS "")
+else()
+  set(WARNING_OPTIONS
+        -Wall
+        -Wextra
+        -Wold-style-cast
+        -Wno-missing-braces
+        -Wcast-align
+        -Wcast-qual
+        -Wctor-dtor-privacy
+        -Wdisabled-optimization
+        -Wformat=2
+        -Winit-self
+        -Wmissing-include-dirs
+        -Woverloaded-virtual
+        -Wredundant-decls
+        -Wshadow
+        -Wsign-promo
+        -Wstrict-overflow=5
+        -fdiagnostics-show-option
+        -Wconversion
+        -Wpedantic
+        -Wno-gnu-zero-variadic-macro-arguments
+        -Wundef
+        -Werror
+        )
+endif(WIN32)
```

Comment:
Modern cmake style is to not put anything in the endif argument

Reply:
Yup - `endif()`